### ISSUE TITLE
Em-dash seems too short

### ISSUE THREAD

**Issue by fontu** [alerque/libertinus#388]
The em-dash seems too short. Can it be longer?
![d](https://user-images.githubusercontent.com/72780791/95844061-6ae1ec80-0d83-11eb-839d-88702f695504.PNG)

**Comment by alerque** [comment]
Yes the emdash is too short. Traditionally the emdash is about the width of an m (I'll have to research whether capital or lower case is more suitable for this measurement) and the endash is closer to an n. Our endash is pretty close to that already but the emdash falls a bit short.

![image](https://user-images.githubusercontent.com/173595/95847971-1411ee00-0d56-11eb-9d78-24a396e5b998.png)

That being said this is going to be a significant change. We will have to be careful about when we release this and how it is versioned. A lot of the other changes I've been making are visually more significant but since they are marks above and below glyphs, etc. changing them will not cause documents to reflow. This change will. In my own case it will cause whole books I've typeset to come out typeset with different page numbers!

**Comment by hvoss49** [comment]
See https://en.wikipedia.org/wiki/Dash#Em_dash  

It is by definition of width 1em, which is, of course not often used in fonts. It looks too wide.

**Comment by georgd** [comment]
One could probably start flame-wars around the correct length of an em-dash. Is it 1em, the width of an `m` or of an `M`...? Fonts exist that do it shorter, some longer. The TNR seems to confuse em-dash with quotation-dash — too long, too thin. Times’ em-dash is shorter but very thin, too. Compared with others, the one of Helvetica looks quite short too, but it’s well aligned with its `M`. In Libertinus, it seems that Mr. Poll oriented the em-dash’s length at the width of the `M`'s top, instead of its bottom. IMO, it doesn’t look as imbalanced as TNR’s em-dash.

Anyway, please be careful with changing this. It might do more harm than good. Perhaps add a longer version as a stylistic variant or character variant?

**Comment by alerque** [comment]
Thanks for the idea on variants @georgd. This *might* be an okay use case for one. I was particularly amenable to this change because I have previously (years ago even) disliked how short is was in my own publishing work.

That being said as I already mentioned I will be careful about the change as this will be a reflow issue. At the very least I'll be batching up changes that can cause reflows. The side bearings on single quotes (see #323) is another one that's going to be a big deal to release. I might try keeping around a long running development branch that has only breaking changes in it and making most changes to master and saving up the breaking ones for a major release when they accrue a bit. I the mean while people really anxious for it could run the development builds (as they can now for every commit to master or PRs).

**Comment by hvoss49** [comment]
@georgd: It has nothing to do with a "flame-war". It was a definition by the typographers in the 20th century or earlier. In the TeX world it is often compared with the _visual_ width of the uppercase M. The _visual_  width is smaller than the box width.

**Comment by georgd** [comment]
@hvoss49 this wasn't directed at you, I’m sorry if it looked so. I had written that before I read your answer and it was meant as a general remark on how eagerly and dogmatically people like to argument on things that are at least partially subject to design choices that can’t be accurately measured.

**Comment by Linguista** [comment]
I would just add that the current em dash is _far_ too short for its use in Spanish, which is to separate parenthetical ideas or introduce dialog. The Times New Roman em dash mentioned above is just about perfect for Spanish use. The Libertinus one looks like an en dash unless you have the two side by side. And in fact, I have had multiple people tell me I used the wrong dash in Spanish texts that use Libertinus.

**Comment by KrasnayaPloshchad** [comment]
> I would just add that the current em dash is far too short for its use in Spanish, which is to separate parenthetical ideas or introduce dialog.
So as a workaround, you can try horizontal line instead.

**Comment by Linguista** [comment]
@KrasnayaPloshchad The horizontal line (U+2015) turned out to be just perfect. Thanks!

**Comment by stuff-and-nonsense** [comment]
Personal names in poetry dedications are sometimes written as the first letter of the name and then a long dash (for privacy). With the current em-dash being too short, you cannot use two or more of them to simulate this.

I first tried the two-em dash (U+2E3A) and three-em dash (U+2E3B), but they are missing.

( Thank you for a libre type with true small caps! )
